eofconvention.fsx

```diff
@@ -0,0 +1,29 @@
+open System.IO
+
+let EOFIsEOL(path: FileInfo) = 
+    try
+        use sr = new StreamReader (path.FullName)
+        let filetext = sr.ReadToEnd()
+        (filetext |> Seq.last = '\n')
+        
+    with
+        | :? System.ArgumentException -> printfn "%A" path; true
```

Comment:
@realmarv ok, getting there, now the next thing to do here is identify which line exactly of those 3 lines inside the `try` block is the one causing ArgumentException, and only put 1 line inside it instead of 3

Reply:
I removed the try with block if it's ok. replaced it with if else

Reply:
nice, much better